.travis.yml

```diff
@@ -30,7 +30,8 @@ matrix:
     - php: 7
       env:
         - DEPS=locked
-        - CHECK_CS=true
+        - CS_CHECK=true
```

Comment:
The variable was wrong!! Please see line 71/73.

Yeah, I know... the best if we have it in separate PR...